open-source-policy/practice.md

```diff
@@ -96,118 +95,111 @@ public.
-    keys, TLS keys, SSH keys, OAuth secrets, or any other “secret keys” that
-    protect access to something.
+  keys, TLS keys, SSH keys, OAuth secrets, or any other “secret keys” that
+  protect access to something.
   * **Undisclosed vulnerabilities:** If we know of a security problem or
-    potential security problem with our code that isn’t already publicly-known
-    (such as a vulnerability that can’t be found with a publicly-available
-    open source scanning tool run on the public-facing system), we shouldn’t
-    write publicly about it until we fix it.
+  potential security problem with our code that isn’t already publicly-known
+  (such as a vulnerability that can’t be found with a publicly-available open
+  source scanning tool run on the public-facing system), we shouldn’t write
+  publicly about it until we fix it.
 * Nonpublic information in general about vulnerabilities, including
-  attribution/source information (such as how and when we learned about a
-  vulnerability, if the disclosure to us was not public).
-* We may wish to withhold some non-CISA IP addresses. If something looks like an
-  IP address, ask CISA Infrastructure before publishing that info.
-* Personally Identifiable Information (PII). Here’s [OMB's definition and GSA's
-  policy](http://www.gsa.gov/portal/content/104256). 18-F also has [guidance for
-  systems involving PII](https://pages.18f.gov/before-you-ship/security/pii/).
+attribution/source information (such as how and when we learned about a
+vulnerability, if the disclosure to us was not public).
+* We may wish to withhold some non-CISA IP addresses. If something looks like
+an IP address, ask CISA Infrastructure before publishing that info.
+* Personally Identifiable Information (PII). Here’s [OMB's definition and GSA's policy](http://www.gsa.gov/portal/content/104256).
+18F also has [guidance for systems involving PII](https://pages.18f.gov/before-you-ship/security/pii/).
 * Some kinds of procurement and acquisition information, which may include
-  non-public cost or pricing data, contract information, trade secrets, indirect
-  costs, and direct labor rates. If you’re an CISA team member working with this
-  kind of data, ask our acquisition specialists for help determining whether it
-  can be public.
+non-public cost or pricing data, contract information, trade secrets, indirect
+costs, and direct labor rates. If you’re an CISA team member working with this
+kind of data, ask our acquisition specialists for help determining whether it
+can be public.
 * Emergency procedures, such as evacuation plans.
 
 There are more categories of controlled unclassified information to protect;
-those are just the kinds that we work with most often. [Here’s the complete
-list.](https://www.archives.gov/cui/registry)
+those are just the kinds that we work with most often. [Here’s the complete list.](https://www.archives.gov/cui/registry)
```

Comment:
I'd move the period outside the link, but ignore if you disagree.

Reply:
I agree, and now the linter allows punctuation after a link without complaining, which is probably why Mark had it inside the link to start with.

Reply:
The page had also moved. I made the link more descriptive in line with the plain language guides and fixed the link in a848650149665216a068543dd1b50962d7ec2737.